### ISSUE TITLE
plotting error dinov2-classification

### ISSUE THREAD

**Issue by c1scott** [roboflow/notebooks#155]
### Search before asking

- [X] I have searched the Roboflow Notebooks [issues](https://github.com/roboflow-ai/notebooks/issues) and found no similar bug report.

### Notebook name

Dinov2-classification
https://github.com/roboflow/notebooks/blob/main/notebooks/dinov2-classification.ipynb

### Bug

Running 
input_file = "MIT-Indoor-Scene-Recognition-5/test/elevator/elevator_google_0053_jpg.rf.41487c3b9c1690a5de26ee0218452627.jpg"

new_image = load_image(input_file)

%matplotlib inline
sv.plot_image(image=new_image, size=(16, 16))

with torch.no_grad():
    embedding = dinov2_vits14(new_image.to(device))

    prediction = clf.predict(np.array(embedding[0].cpu()).reshape(1, -1))

    print()
    print("Predicted class: " + prediction[0])

i got

---------------------------------------------------------------------------
error                                     Traceback (most recent call last)
[<ipython-input-21-0caa780c49bc>](https://localhost:8080/#) in <cell line: 6>()
      4 
      5 get_ipython().run_line_magic('matplotlib', 'inline')
----> 6 sv.plot_image(image=new_image, size=(16, 16))
      7 
      8 with torch.no_grad():

[/usr/local/lib/python3.10/dist-packages/supervision/utils/notebook.py](https://localhost:8080/#) in plot_image(image, size, cmap)
     33         plt.imshow(image, cmap=cmap)
     34     else:
---> 35         plt.imshow(cv2.cvtColor(image, cv2.COLOR_BGR2RGB))
     36 
     37     plt.axis("off")

error: OpenCV(4.7.0) :-1: error: (-5:Bad argument) in function 'cvtColor'
> Overload resolution failed:
>  - src is not a numpy array, neither a scalar
>  - Expected Ptr<cv::UMat> for argument 'src'
<Figure size 1600x1600 with 0 Axes>

### Environment

Google Colab, GPU T4 runtime

### Minimal Reproducible Example

run all the cells (authorize roboflow when prompted), when you get to the last runnable cell

input_file = "MIT-Indoor-Scene-Recognition-5/test/elevator/elevator_google_0053_jpg.rf.41487c3b9c1690a5de26ee0218452627.jpg"

new_image = load_image(input_file)

%matplotlib inline
sv.plot_image(image=new_image, size=(16, 16))

with torch.no_grad():
    embedding = dinov2_vits14(new_image.to(device))

    prediction = clf.predict(np.array(embedding[0].cpu()).reshape(1, -1))

    print()
    print("Predicted class: " + prediction[0])
    
    if you run it you will see the error reproduced
    however, the classification works and is correct

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SkalskiP** [comment]
Hi @c1scott 👋🏻 Let me take a look.

**Comment by SkalskiP** [comment]
@capjamesg could you take a look at this notebook? I did some initial investigation. It looks like `new_image` in this line:

```python
new_image = load_image(input_file)
```

It is a PyTorch tensor, but you pass it to `plot_image`. `plot_image` can only take Numpy arrays.

**Comment by capjamesg** [comment]
@c1scott Thank you for the report! I have filed a PR to fix the notebook.

In any case, you can use this code to show the image:

```python
%matplotlib inline
sv.plot_image(image=cv2.imread(input_file), size=(8, 8))
```

This will open the image with `cv2` and display it using `supervision`.

**Comment by capjamesg** [comment]
This issue was fixed in PR #157 so I am closing this issue.
